app/main.py

```diff
@@ -17,3 +17,14 @@
 collection_of_coins = {1, 2, 25}
 
 # write your code here
+sorted_variebles = {
+    "mutable" : [],
+    "immutable" : []
```

Comment:
The variable name `immutable` is correctly used here, but ensure consistency in naming conventions.